src/stores/Tables/Store.ts

```diff
@@ -340,6 +354,35 @@ export const getInitialState = (
             );
         },
 
+        setActionSettings: (setting, values, addOperation) => {
+            set(
+                produce((state: SelectableTableStore) => {
+                    const existingEl = state.actionSettings[setting] ?? [];
+
+                    if (addOperation) {
+                        state.actionSettings[setting] =
+                            existingEl.concat(values);
+                    } else if (existingEl.length > 0) {
+                        state.actionSettings[setting] = existingEl.filter(
+                            (el) => !values.includes(el)
+                        );
+                    }
```

Comment:
Just thinking... I think we'd probably want to "invert" this and have a loop go over all the `values` passed in. This does not matter right now because it is always a single element array... but in the future if we allow a "Select All" functionality I think looping over just the array that includes changes should be best.